scripts/build-newsroom-videos.js

```diff
@@ -0,0 +1,39 @@
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const fetch = require('node-fetch')
+
+async function buildNewsroomVideos () {
+    
+    try{
+        let data;
+        const response = await fetch('https://youtube.googleapis.com/youtube/v3/search?' + new URLSearchParams({
+          key: process.env.YOUTUBE_TOKEN,
+          part: 'snippet',
+          channelId: 'UCIz9zGwDLbrYQcDKVXdOstQ',
+          eventType: 'completed',
+          type:'video',
+          order: 'Date',
+          maxResults: 5,
+        }))
+        data = await response.json()
+        const videoDataItems = data.items.map((video) =>{
+            return {
+                image_url:video.snippet.thumbnails.high.url,
+                title: video.snippet.title,
+                description: video.snippet.description,
+                videoId: video.id.videoId,
+            }
+        })
+        const videoData = JSON.stringify(videoDataItems, null, '  ');
+        console.log('THe following are the Newsroom Youtube videos', videoData)
+
+        writeFileSync(
+            resolve(__dirname, '../config', 'newsroom_videos.json'),
+            videoData
+        );
+    }catch(err){
+        console.log(err)
```

Comment:
I was thinking, cause already when building roadmap, we throw in a face of developer some strange fetch error, which is related to lack of token. but contributors that start do not know it.

Here we will have the same. Maybe we should do that on CI we have another env maybe, like `process.env.PRODUCTION` and when we pass it in the CI, we run `console.log(err)` and otherwise we run `console.log('Skipping building of a current list of YouTube videos for the newsroom section as token was not provided. This is what we do on local in dev environment. Keep calm 😄')`

what do you think?

Reply:
But this particular only runs when the workflow runs in the morning and only on the `asyncapi/website` repository, not on any forks. Then, I don't think anyone will get strange errors while starting on the development mode of the website.

Reply:
omg, you are right 🤦🏼 forgot it is the same as with meetings, it is not part of the build process